tests/microos/toolbox.pm

```diff
@@ -0,0 +1,85 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run simple toolbox tests
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use version_utils 'is_sle_micro';
+
+
+sub run {
+    select_console 'root-console';
+    my $user = 'test_user';
+
+    # Create user and don't ask password for sudo commands
```

Comment:
Should be deleted at the end IMO

Reply:
Good point. Just to leave the system as it was.